udata/migrations/2015-11-16-remove-supplier.js

```diff
@@ -0,0 +1,11 @@
+/**
+ * Remove supplier property from datasets
+ */
+function process_collection(collection) {
+    collection.update({supplier: {$exists: true}},
+                                 {$unset: {'supplier': 1}}, false, true);
+    var count = db.runCommand({getLastError : 1}).n;
```

Comment:
You can also use the update return value which contain `nModified`. The current pattern will only work if there was no other operation in the interval.